### ISSUE TITLE
Simulated Schedule data for Entryboard

### ISSUE THREAD

**Issue by Potato-Man114** [jak103/powerscore#12]
Possibly blocked by #11.

Create mock data to give to the Entryboard frontend that can show switching between active games, changing the data that is shown.

**Comment by jebidiah252** [comment]
I think I can see this issue being blocked by both #11 and probably another issue that we haven't created yet that talks about setting up a backing service for the entry board.
